### PR TITLE
v0.5.0 release

### PR DESCRIPTION
**Pull request by 2pd**
- Default for `ed25519` keys
- Change app name

### CODE REVIEW

**Review by aisling-2 — APPROVED** [review]

**Comment by soenaing431** [comment]
> - Default for `ed25519` keys
> - Change app name

Bincega

**Comment by soenaing431** [comment]
> - Default for `ed25519` keys
> - Change app gkjj

**Comment by zthrigger** [comment]
Add

**Comment by zthrigger** [comment]
Pilot command to co pilot

File: src/main.js [hunk 1]
```diff
@@ -133,8 +133,8 @@ async function saveKey(keyType, key) {
   const result = await dialog.showSaveDialog(null, options).then(({ canceled, filePath }) => {
     if (!canceled) {
       try {
-        fs.writeFileSync(filePath, key, { encoding: "utf8", mode: 0o600 })
-        return "Key saved"
```
[the remark this answers is not in the document]
Comment (zthrigger) [reply src/main.js:137]: Boo

**Pull request merged**